Makefile

```diff
@@ -8,8 +8,17 @@ export PROJECT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 export ADI_DIR:=${PROJECT_DIR}/adi
 export ROOT_DIR:=$(shell dirname "${PROJECT_DIR}")
 
+.PHONY: build test
+
 build:
-	bash ${ADI_DIR}/build.sh
+	echo "${VERSION}" > VERSION.txt
+	pip install -r ${PROJECT_DIR}/requirements/test.txt
 
 test:
 	bash ${ADI_DIR}/test.sh
+
+release:
+	bash ${ADI_DIR}/release.sh
+
+version:
+	echo "1.0.${BUILD_IDENTIFIER}"
```

Comment:
I think it's worth mentioning in the docs, considering this is opensource and open to contributions. So building locally with `make build` without having been careful has its risks.

Options:
- Check the value of env var `$VIRTUAL_ENV` and fail if not defined, with some message. This is compatible with `pyenv` (I've also now switched to this) and regular `virtualenv`.
- Migrate to use `tox` to build the builder for both Python 2 and 3.